app-creator/client/src/widgets/templates-tab/test/templates-tab_test.ts

```diff
@@ -14,4 +16,56 @@ suite('templates-tab', () => {
     const el = await fixture(html`<templates-tab></templates-tab>`);
     expect(el.shadowRoot!.childNodes.length).to.be.greaterThan(0);
   });
+
+  suite('filter template cards', () => {
+    test('Empty query from all devices', () => {
+      const result = TemplatesTab.filterTemplates(
+        templatesStub,
+        '',
+        DeviceType.all
+      );
+      expect(result.length).to.equal(templatesStub.length);
```

Comment:
Thanks for adding these tests, they are great! I would recommend renaming this tests "Empty query for all device types" and enumerate the two other `DeviceType`s for completeness.